File: Python/01-Arrays/Ejercicio2.py
# Dado un vector de enteros, comprobar el mayor, el menor y por último la media de todos. La lista 
# termina cuando el numero sea negativo o Cero

numeros=[]
i=0
suma=0

while True:
  numero=int(input("Ingrese un numero: "))

  if numero>0:
    i+=1
    suma+=numero
    numeros.append(numero)

  else:
    print("Los numero ingresados fueron: ", numeros)
    media=suma/i
    numeros.sort()
    print("El numero menor es: ", numeros[0] )
    print("El numero menor es: ", numeros[i-1] )
    print("La media de los numero ingresados es: ", media)
    break


  

File: JavaScript/Ejercicios-fundamentos/Ejercicio2.js
// 2.	Pedir 10 sueldos. Mostrar su suma y cuantos
//  hay mayores de 1000€


let i=0;
let sueldo=0;
let suma=0;
let mayor=0;
let menor=0;


while (i <= 10) {
    sueldo=parseInt(prompt("Ingrese su sueldo: "));
    suma+=sueldo;
    i++
    if(sueldo>=1000){
        mayor+=1
    }
    else{
        menor+=1
    }
}

console.log(`La suma de todos los sueldos es: ${suma}, y hay ${mayor} sueldos mayores a 1000€`);
